test/functional/wallet_txn_clone.py

```diff
@@ -90,10 +90,13 @@ def run_test(self):
         # Node0's balance should be starting balance, plus 50TPC for another
         # matured block, minus tx1 and tx2 amounts, and minus transaction fees:
         expected = starting_balance + node0_tx1["fee"] + node0_tx2["fee"]
+        coinbase = {}
         if self.options.mine_block:
-            expected += 50
-        expected += tx1["amount"] + tx1["fee"]
-        expected += tx2["amount"] + tx2["fee"]
+            block = self.nodes[0].getblock(self.nodes[0].getbestblockhash())
+            coinbase = self.nodes[0].gettransaction(block['tx'][0])
+            expected += coinbase['amount']
```

Comment:
```py
# Node0's balance should be starting balance, plus 50TPC for another
# matured block, minus tx1 and tx2 amounts, and minus transaction fees:
```

If you test according to this test scenario, don't you need Coinbase transactions and can leave them as they are?

Reply:
sorry. I missed this as next comment is also related to the same node balance. In our test case while mining a block the non maleated transactions are confirmed and add fee also to the coinbase amount. I'll update the comments.

Reply:
Are the fees different from those added in the code below?

```
expected += (tx1["amount"] + tx1["fee"])
expected += (tx2["amount"] + tx2["fee"])
```

Reply:
This test runs in two modes. When 'mine_block' is enabled, tx1 and tx2 get mined and confirmed. In the default mode these two remain in the mempool. This is why there is a difference in the balances. In bitcoin, mine_block mode does not execute the test till the end. It just verifies the clones not mined midway and exits.

Reply:
I updated the calculations again. Hope its cleaner and clearer now.